packages/components/table/src/styled_tbody.tsx

```diff
@@ -16,19 +16,7 @@ export const StyledTbody = styled.tbody<StyledTbodyProps>`
         cursor: pointer;
 
         &:hover {
-          background-color: ${({ theme }) => theme.colors.cloud};
-        }
-
-        &:focus {
-          background-color: ${({ theme }) => theme.colors.white};
-          outline: none;
-        }
-
-        &:active {
-          outline-color: ${({ theme }) => theme.colors.blue};
-          outline-offset: -3px;
-          outline-style: solid;
-          outline-width: 3px;
+          background-color: ${({ theme }) => theme.colors.background.hover};
```

Comment:
Just wondering about the key being used here. Is there a single hover background color for all things?

Reply:
It's part of the figma design tokens for colors as defined here

<img width="782" alt="image" src="https://user-images.githubusercontent.com/105694/217572624-adcdeb79-44fa-4ca7-95df-4cdb2a5b3da6.png">